tests/plugins/tableselection/manual/mouseover.md

```diff
@@ -0,0 +1,20 @@
+@bender-ui: collapsed
```

Comment:
File name rather than `mouseover.md` should also mention scroll, so `scrollmouseover.md`?